UI-Session-4-Assignment/index.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
```

Comment:
you have used only header, section and footer semantics tag, try to use more semantic tags.

Reply:
Done.
Used main tag.
Any other semantic tag doesn't fit in this assignment.